src/MCT.js

```diff
@@ -95,13 +95,6 @@ define([
      */
     function MCT() {
         EventEmitter.call(this);
-        this.buildInfo = {
```

Comment:
This should not be deleted. It's being used elsewhere. Is eslint complaining about the various "`__`" variables? If so, either add them all as globals to this file, or surround this block with an exception. They are injected by Webpack as string literals.